travis/build-docker-images.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+# Only push the docker container to Docker Hub for master branch
+if [[ "$BRANCH" == "master" && "$TRAVIS_SECURE_ENV_VARS" == "true" ]]; then echo 'upload to Docker Hub'; else echo 'skip docker upload for PR'; exit 0; fi
+
+source ~/.nvm/nvm.sh
+nvm use 6
+DOCKER_NAMESPACE=jaegertracing make docker
```

Comment:
Not really. In bash, this has the effect of setting this var for this command, so, `make` will see DOCKER_NAMESPACE as `jaegertracing` without the parent script needing to `export` it first.